src/data/science.json

```diff
@@ -30,6 +31,7 @@
       "url": "/de/science",
       "external": false
     },
+    "mobileMessage": "Die Seite ist für Mobilgeräte optimiert, d.&nbsp;h. Grafiken werden als statische Bilder angezeigt.",
```

Comment:
Maybe "Diese Seite" instead of "Die Seite"?
@Ein-Tim 
What do you think as native German speaker?

Reply:
Yes, "Diese" is better 👍

Reply:
Thanks.